docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-REST-API-configuration-ref.adoc

```diff
@@ -0,0 +1,419 @@
+[id='optimizer-REST-API-configuration-ref']
+= Configure the {PLANNER} REST API
+//Not sure how to test most of the examples here to demonstrate the Employee Rostering example snippets. Require SME input.
+
+The {KIE_SERVER} supports the following {PLANNER} REST APIs.
+These APIs are also available through JMS and the Java client API.
+
+.Prerequisite
+{PRODUCT} has been installed and configured with basic HTTP Authentication for the `kie-server` role. You have identified the server endpoint base URL, for example, `http://SERVER:PORT/CONTEXT/services/rest/server/`.
+
+. To get a specific marshalling format, add the HTTP headers `Content-Type` and optional `X-KIE-ContentType` in the HTTP request. For example:
+
+[source]
+----
+Content-Type: application/xml
+X-KIE-ContentType: xstream
+----
+
+[NOTE]
+====
+`X-KIE-ContentType` supports the following values: `xstream`, `xml`, `json`.
+====
+
+The requests and responses in the example below assume that a KIE container is built using the Employee Rostering example in {CENTRAL}, by calling a `PUT` on `/services/rest/server/containers/employee-rostering` with this content:
+
+[source,xml]
+----
+<kie-container container-id="employee-rostering">
+  <release-id>
+    <group-id>employeerostering</group-id>
+    <artifact-id>employeerostering</artifact-id>
+    <version>1.0.0-SNAPSHOT</version>
+  </release-id>
+</kie-container>
+----
+
+== [GET] /containers
+
+
+Returns the list of created containers.
+
+.Example Server Response (XStream)
+====
+[source,xml]
+----
+<response type="SUCCESS" msg="List of created containers">
+  <result class="kie-containers">
+    <kie-container>
+      <container-id>employee-rostering</container-id>
+        <release-id>
+          <group-id>employeerostering</group-id>
+          <artifact-id>employeerostering</artifact-id>
+          <version>1.0.0-SNAPSHOT</version>
+        </release-id>
+        <resolved-release-id>
+          <group-id>employeerostering</group-id>
+          <artifact-id>employeerostering</artifact-id>
+          <version>1.0.0-SNAPSHOT</version>
+        </resolved-release-id>
+        <status>STARTED</status>
+        <scanner>
+          <status>DISPOSED</status>
+        </scanner>
+    </kie-container>
+  </result>
+</response>
+----
+====
+
+
+== [PUT] /containers/{containerId}/solvers/{solverId}
+
+Creates a new solver with the given `{solverId}` in the container ``{containerId}``.
+The request's body is a marshalled SolverInstance entity that must specify the solver configuration file.
+
+The following is an example of the request and the corresponding response.
+
+.Example Server Request (XStream)
+====
+[source,xml]
+----
+<solver-instance>
+  <solver-config-file>optacloud/optacloud/cloudSolverConfig.solver.xml</solver-config-file>
+</solver-instance>
+----
```

Comment:
Yes please! I did have a comment in the original PR for this for the right request and responses for ER, but I had to remove comments when we needed to merge and build the books. 

I'd prefer to refer to the employee-rostering sample wherever possible since it's the one we expect customers to want to use the most.